services/m3aggregator/config/aggregator.go

```diff
@@ -124,7 +125,7 @@ type AggregatorConfiguration struct {
 	MaxFlushSize int `yaml:"maxFlushSize"`
 
 	// Flushing handler configuration.
-	Flush flushHandlerConfiguration `yaml:"flush"`
+	Flush []flushHandlerConfiguration `yaml:"flush"`
```

Comment:
I think we should keep this as `flushHandlerConfiguration` and define a separate `broadcastHandlerConfiguration` which contains a list of `flushHandlerConfiguration`, i.e., the config will look like:
```
flush:
    type: broadcast
    broadcast:
      - type: forward
         forward:
           ...
       - type: forward
           ...
```
This makes it more flexible to configure the handlers, and doesn't force the users to always broadcast if they only have a single handler.